Fix from upstream:

Fix hanging session on close (leading to an increasing
number of krusader instances on KDE start).

Index: krusader/panelmanager.cpp
===================================================================
--- krusader/panelmanager.cpp	(revision 1137335)
+++ krusader/panelmanager.cpp	(revision 1154581)
@@ -121,6 +121,7 @@
 
 //     _stack->setUpdatesEnabled(false);
     _stack->setCurrentWidget(_self);
+    kapp->processEvents();
     // make sure the view is focused (this also causes ListPanel::slotFocusOnMe() to be called)
     _self->view->widget()->setFocus();
 //     _stack->setUpdatesEnabled(true);
